app/styles/layout/calendar.css

```diff
@@ -0,0 +1,72 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.fc-toolbar {
+  @apply border border-gray-300 p-2 m-0 !important;
+  border-bottom: none !important;
+  border-top-left-radius: 0.5rem !important;
+  border-top-right-radius: 0.5rem !important;
+}
+
+.fc-event-time {
+  @apply hidden !important;
+}
+
+.fc-event-title {
+  @apply ml-1 !important;
+}
+
+.fc-next-button {
+  @apply ml-0 rounded-tl-none rounded-bl-none border border-gray-400 bg-white text-gray-600 !important;
+}
+
+.fc-today-button {
+  @apply ml-0 rounded-none border border-gray-400 bg-white text-black pt-[0.5em] !important;
+  border-left-width: 0 !important;
```

Comment:
Instead of doing this you can use border-x or border-y to only add on the axis you want